### PR TITLE
Add tunnelmole to Proxy

### PR DESCRIPTION
**Pull request by WisdomSky**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software) must be added to `non-free.md` and have the license set to `⊘ Proprietary`
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding has working installation instructions.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1653,7 +1653,7 @@ _Related: [Web Servers](#web-servers)_
 - [Squid](http://www.squid-cache.org/) - Caching proxy for the Web supporting HTTP, HTTPS, FTP, and more. It reduces bandwidth and improves response times by caching and reusing frequently-requested web pages. ([Source Code](https://code.launchpad.net/squid)) `GPL-2.0` `C`
 - [SWAG (Secure Web Application Gateway)](https://github.com/linuxserver/docker-swag) - Nginx webserver and reverse proxy with PHP support, built-in Certbot (Let's Encrypt) client and fail2ban integration. `GPL-3.0` `Docker`
 - [Tinyproxy](https://tinyproxy.github.io/) - Light-weight HTTP/HTTPS proxy daemon. ([Source Code](https://github.com/tinyproxy/tinyproxy)) `GPL-2.0` `C`
-
+- [Tunnel Mole](https://tunnelmole.com/) - Tunnelmole is a simple tool to give your locally running web applications a public URL. Opensource alternative to ngrok. ([Source Code](https://github.com/robbie-cahill/tunnelmole-service)) `AGPL-3.0` `Nodejs`
```
Comment (KieranRobson) [inline README.md:1656]:
```suggestion
- [Tunnel Mole](https://tunnelmole.com/) - A simple tool to give your locally running web applications a public URL. Open source alternative to ngrok. ([Source Code](https://github.com/robbie-cahill/tunnelmole-service)) `AGPL-3.0` `Nodejs`
```

More concise description

**Review by KieranRobson — CHANGES_REQUESTED** [review]
A recommended description change.

**Review by kokomo123 — CHANGES_REQUESTED** [review]
Please remove the extra space and add your entry next to tinyproxy, and i think that should be good to go.

**Comment by WisdomSky** [comment]
> Please remove the extra space 

May I know which extra spaces you were referring to? 

> and add your entry next to tinyproxy

It's currently next to tinyproxy.

**Comment by nodiscc** [comment]
Please re-add the deleted blank line below your new entry

**Comment by kokomo123** [comment]
What nodiscc has said, apologies. I thought it was the blank space next to tinyproxy, but its a blank line below.

**Comment by WisdomSky** [comment]
@nodiscc @kokomo123 

Done. Please check.

**Comment by nodiscc** [comment]
Sorry, this contribution guideline was added in the mean time:

> https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/.github/PULL_REQUEST_TEMPLATE.md
>  Any software project you are adding was first released more than 4 months ago.

So this does not qualify. Please resubmit 4 months after the first stable release (there are no releases yet).

**Pull request closed without merging**